pkg/block/fetcher.go

```diff
@@ -170,6 +170,51 @@ func DefaultModifiedLabelValues() [][]string {
 	}
 }
 
+type BlockIDsFetcher interface {
+	GetActiveAndPartialBlockIDs(ctx context.Context) (metaIDs[]ulid.ULID, partialBlocks map[ulid.ULID]bool, err error)
```

Comment:
Need comments on the new interface and its method

Reply:
Will add a description.